scripts/check_overall_backend_test_coverage_test.py

```diff
@@ -0,0 +1,108 @@
+# Copyright 2022 The Oppia Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Unit tests for scripts/check_overall_backend_test_coverage.py."""
+
+from __future__ import annotations
+
+import builtins
+import subprocess
+import sys
+
+from core.tests import test_utils
+from scripts import check_overall_backend_test_coverage
+from typing import Any
+
+
+class CheckOverallBackendTestCoverageTests(test_utils.GenericTestBase):
+    """Unit tests for scripts/check_overall_backend_test_coverage.py."""
+
+    def setUp(self) -> None:
+        super().setUp()
+
+        self.print_arr: list[str] = []
+        def mock_print(msg: str, end: str = '\n') -> None:  # pylint: disable=unused-argument
+            self.print_arr.append(msg)
+        self.print_swap = self.swap(builtins, 'print', mock_print)
+
+    def test_no_data_in_coverage_report_throws_error(self) -> None:
+        class MockProcess:
+            returncode = 0
+            stdout = 'No data to report.'
+            stderr = 'None'
+        def mock_subprocess_run(*args: Any, **kwargs: Any) -> MockProcess: # pylint: disable=unused-argument
+            return MockProcess()
+        swap_subprocess_run = self.swap(subprocess, 'run', mock_subprocess_run)
+
+        with swap_subprocess_run, self.assertRaisesRegex( # type: ignore[no-untyped-call]
+            RuntimeError,
+            'Run backend tests before running this script. ' +
+            '\nOUTPUT: No data to report.\nERROR: None'):
```

Comment:
Fix indentation. Ditto elsewhere

```suggestion
        with swap_subprocess_run, self.assertRaisesRegex( # type: ignore[no-untyped-call]
            RuntimeError,
            'Run backend tests before running this script. ' +
            '\nOUTPUT: No data to report.\nERROR: None'
        ):
```

Reply:
Done